appimage-amd64.yaml

```diff
@@ -0,0 +1,122 @@
+# appimage-builder recipe file https://appimage-builder.readthedocs.io/en/latest/
+version: 1
+
+AppDir:
+  path: ./AppDir
+
+  app_info:
+    id: com.georgefb.haruna
+    name: Haruna
+    icon: com.georgefb.haruna
+    version: 1.0.0
```

Comment:
Why 1.0.0? There is no 1.0.0 version for Haruna. Where is this version used?

Reply:
Sorry, what's the right version ?

Reply:
The latest version is 0.5.0 https://github.com/g-fb/haruna/releases, but if this builds from the master branch then it should be set to something else: master, dev.

Reply:
We could use something like `continuous-<short sha commit id>`

Reply:
> We could use something like continuous-< short sha commit id> for master

That is fine.

> and only the sha commit id for other builds.

Which other builds? If you mean for tags/releases then those should use the actual version.

Reply:
> Which other builds? If you mean for tags/releases then those should use the actual version.

The ones you decide to have as project owner.
Please let me know what else is required for having AppImage packages of haruna.

Reply:
> Please let me know what else is required for having AppImage packages of haruna.

Just change the version to be `continuous-<short sha commit id>`, then I'll merge this and we'll see how it goes.

Reply:
as we will be creating an AppImage of every build I used the branch name instead of continuous this way you can test all PR made to your repository without building it locally. You can also add an additional step to the github workflow to publish the artifact as releases when a commit is made into master.